example/lib/main.dart

```diff
@@ -2383,19 +2377,17 @@ class _MyAppState extends State<MyApp> {
     print("PE -> Sync Variables");
   }
 
-  void fetchVariables() {
+  void fetchVariables() async {
     showToast("Fetch Variables");
-    this.setState(() async {
-      bool? success = await CleverTapPlugin.fetchVariables();
-      print("PE -> fetchVariables result: " + success.toString());
-    });
+    bool? success = await CleverTapPlugin.fetchVariables();
+    print("PE -> fetchVariables result: " + success.toString());
   }
 
   void defineVariables() {
     var variables = {
       'flutter_var_string': 'flutter_var_string_value',
-      'flutter_var_map': {'flutter_var_map_string': 'flutter_var_map_value'},
-      'flutter_var_int': 6,
+      // 'flutter_var_map': {'flutter_var_map_string': 'flutter_var_map_value'},
+      'flutter_var_int': 'sonam',
```

Comment:
@KambleSonam int variable is being defined as String

Reply:
Corrected this.